src/Desktop.Analyzers/Core/DoNotUseInsecureCryptographicAlgorithms.cs

```diff
@@ -55,6 +55,11 @@ private static DiagnosticDescriptor CreateDiagnosticDescriptor(string ruleId, Lo
 
         public override void Initialize(AnalysisContext analysisContext)
         {
+            analysisContext.EnableConcurrentExecution();
+
+            // Security analyzer - analyze and report diagnostics in generated code.
```

Comment:
for generated code, once user has diagnostic reported for the file, except suppressing it, what is users options to fix the diagnostic?